external/fv3fit/fv3fit/keras/_models/shared/pure_keras.py

```diff
@@ -37,9 +36,8 @@ def __init__(
         self,
         input_variables: Iterable[Hashable],
         output_variables: Iterable[Hashable],
-        output_metadata: Iterable[Dict[str, Any]],
         model: tf.keras.Model,
-        unstacked_dims: Optional[Sequence[str]] = None,
+        unstacked_dims: Sequence[str] = ("z",),
```

Comment:
See above about whether this should have a default.

Reply:
Updated to required argument.